### ISSUE TITLE
Positive Phalen test

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6727]
**Preferred term label:**
Positive Phalen test

**Synonyms**
Phalen maneuver

**Definition (free text, please give PubMed ID)**
The Phalen maneuver is performed by having the patient hold both wrists in complete and forced flexion (pushing the dorsal surfaces of both hands together) for 30–60 seconds. This can increase the pressure in the carpal tunnel. The test is positive (abnormal) if the patient experiences characteristic symptoms of carpal tunnel syndrome (pain and paresthesias along the distribution of the median nerve, i.e., thumb, index finger, and middle finger).
PMID:31773104

See Figure 2 of PMID:31773104

**Parent term (use hpo.jax.org/app)**
Abnormal peripheral nervous system physiology HP:0032120

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619161

14/14 PMID:32686688

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
